lib/vet360/contact_information/configuration.rb

```diff
@@ -10,7 +10,7 @@ def base_path
       end
 
       def service_name
-        'Vet360'
+        'Vet360/ContactInformation'
```

Comment:
@rlbaker @hpjaj Wyatt and I noticed that this name (in the sub sub class) was referencing the generic service instead of the specific service (ContactInformation)... which would cause weirdness down the road when another Vet360-sub-service exists. I think this change is safe and the tests still pass, but I wanted to flag this to make sure you saw it in case I'm overlooking something.

Reply:
cc @wyattwalter